### PR TITLE
[Hub Generated] Review request for Microsoft.Security to add version stable/2019-08-01

### PR DESCRIPTION
**Pull request by kfbehar**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/kfbehar/azure-rest-api-specs/dev-security-Microsoft.Security-2019-08-01...master/).

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotSecuritySolutions.json [hunk 1]
```diff
@@ -378,6 +378,9 @@
           "type": "string",
           "description": "Workspace resource ID"
         },
+        "additionalWorkspaces": {
```
Comment (ryansbenson) [inline specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotSecuritySolutions.json]: New properties added to the response of an API is considered a breaking change and requires a new API version:

https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo

File: specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotSecuritySolutions.json [hunk 2]
```diff
@@ -636,8 +639,51 @@
...
+      },
+      "region": {
+        "type": "string",
+        "description": "The region that the workspace resides in"
+      },
+      "type": {
+        "type": "string",
+        "description": "The type of the workspace",
+        "enum": [
+          "Sentinel"
```
Comment (ryansbenson) [inline specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotSecuritySolutions.json]: If you only have one enum value for type, does it need to be in the model?

**Comment by MyronFanQiu** [comment]
@kfbehar Hello. ARM team wants this change happened in a new api version.

File: specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotSecuritySolutions.json (hunk 1 above)
Comment (kfbehar) [reply specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotSecuritySolutions.json]: Our SDK and REST docs are not published yet (will be in the near future), so customers aren't supposed to make calls to this API yet.

File: specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotSecuritySolutions.json (hunk 2 above)
Comment (kfbehar) [reply specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotSecuritySolutions.json]: Currently the only option is Sentinel, but we wanted to future-proof this.

**Comment by MyronFanQiu** [comment]
@kfbehar Please share your thoughts about this PR and ARM's comments.

**Comment by MyronFanQiu** [comment]
@kfbehar kindly ping.

**Comment by MyronFanQiu** [comment]
@kfbehar hi. is there any plan for this PR?

**Comment by MyronFanQiu** [comment]
@kfbehar Hello. If you don't need this PR anymore, I will close this PR in this week.

File: specification/security/resource-manager/Microsoft.Security/preview/2020-02-01-preview/iotSecuritySolutions.json [hunk 3]
```diff
@@ -0,0 +1,734 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2019-08-01"
```
Comment (MyronFanQiu) [inline specification/security/resource-manager/Microsoft.Security/preview/2020-02-01-preview/iotSecuritySolutions.json]: should it be 2020-02-01-preview?

File: specification/security/resource-manager/Microsoft.Security/preview/2020-02-01-preview/iotSecuritySolutions.json [hunk 4]
```diff
@@ -0,0 +1,734 @@
...
+    },
+    "RecommendationConfigurationList": {
+      "type": "array",
+      "description": "List of the configuration status for each recommendation type.",
+      "items": {
+        "$ref": "#/definitions/RecommendationConfigurationProperties"
+      }
+    },
+    "AdditionalWorkspaceProperties": {
+    "type": "object",
```
Comment (MyronFanQiu) [inline specification/security/resource-manager/Microsoft.Security/preview/2020-02-01-preview/iotSecuritySolutions.json]: could we add `\t` here to have consistent format?

File: specification/security/resource-manager/Microsoft.Security/preview/2020-02-01-preview/iotSecuritySolutions.json [hunk 5]
```diff
@@ -0,0 +1,734 @@
...
+      "workspace": {
+        "type": "string",
+        "description": "The ARM ID of the workspace"
+      },
+      "type": {
+        "type": "string",
+        "description": "The type of the workspace",
+        "enum": [
+          "Sentinel"
+        ]
```
Comment (MyronFanQiu) [inline specification/security/resource-manager/Microsoft.Security/preview/2020-02-01-preview/iotSecuritySolutions.json]: Only the following two fields don't include `x-ms-enum`. Is it on purpose?

**Review by mmyyrroonn — CHANGES_REQUESTED** [review]
LGTM in general. Please fix api-version part.

**Comment by MyronFanQiu** [comment]
@kfbehar Kindly ping

**Comment by MyronFanQiu** [comment]
@kfbehar kindly ping

File: specification/security/resource-manager/Microsoft.Security/preview/2020-02-01-preview/iotSecuritySolutions.json (hunk 5 above)
Comment (kfbehar) [reply specification/security/resource-manager/Microsoft.Security/preview/2020-02-01-preview/iotSecuritySolutions.json]: Opened a new PR with the requested changes
https://github.com/Azure/azure-rest-api-specs/pull/8545

**Review by kfbehar — COMMENTED** [review]
Opened a new PR with the requested changes
https://github.com/Azure/azure-rest-api-specs/pull/8545

**Pull request closed without merging**